<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TESS_API_EXAMPLEClass</class>
 <widget class="QMainWindow" name="TESS_API_EXAMPLEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TESS_API_EXAMPLE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="btnOpenNet">
      <property name="text">
       <string>打开路网</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartSimu">
      <property name="text">
       <string>启动仿真</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPauseSimu">
      <property name="text">
       <string>暂停仿真</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStopSimu">
      <property name="text">
       <string>停止仿真</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>信息窗</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="txtMessage"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TESS_API_EXAMPLE.qrc"/>
 </resources>
 <connections/>
</ui>
